packages/angular-generator/src/expressions/jsx/element_post_processing.ts

```diff
@@ -0,0 +1,16 @@
+import { toStringOptions } from '@devextreme-generator/core';
+import type { JsxOpeningElement } from './jsx-opening-element';
+import { tryToGetContent } from './ng-content-generator';
+
+export const process = (element: JsxOpeningElement, options?: toStringOptions): { prefix: string, postfix: string } => {
+  const { content, elementDirective, condition } = tryToGetContent(element);
+
+  if (elementDirective !== null) {
+    element.attributes.push(elementDirective);
+  }
```

Comment:
Why do we need two same fiiles under defferent names?

Reply:
left over from the refactor, fixed